db/migrate/20160519031755_create_deal_stages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDealStages < ActiveRecord::Migration
```

Comment:
I'll merge this as is - but will update this table and it's class association in a new branch so so we could call something like 
```ruby
Deal.find(32423).deal_stage.human_readable
```